index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
It looks like your editor may have been set to make all tags xhtml compliant (adding the / for the empty tag shorthand). However, the DOCTYPE we are using is html5, which doesn't need (and recommands against) closing tags like this if they are truly void elements. It's possible to put the validator into xhtml mode by using an appropriate DOCTYPE, such as
```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
   "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

Personaly, I do prefer xhtml over html (and JSX syntax is based on xhtml!).